GetDKAN/dkan
# Allow deleting a harvest without deleting its datasets

Proposed fix for [#3839]

### Steps to test

1. Create and run a harvest
2. Verify the harvest is registered, i.e. shown in the output of `drush dkan:harvest:list`
3. Verify its datasets exist and are published
4. Deregister the harvest: `drush dkan:harvest:deregister HARVEST_ID  --revert`
5. Verify the harvest is no longer shown in the output of `drush dkan:harvest:list`
6. Verify its datasets still exist, and are published
